config/config.go

```diff
@@ -48,6 +48,9 @@ type Config struct {
 	// Dedup is used to configure the dedup settings
 	Dedup *DedupConfig `mapstructure:"deduplicate"`
 
+	// Default is used to configure the defaults for templates
+	Defaults *DefaultsConfig `mapstructure:"defaults"`
```

Comment:
Could you rename `Defaults` to something more descriptive... maybe `DefaultDelims` or similar.
Same for the `DefaultsConfig` struct. Please keep the fact that they match.
Thanks.